File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.py
class Solution(object):
    def subtractProductAndSum(self, n):
        """
        :type n: int
        :rtype: int
        """
        temp=n
        sum=0
        product=1
        while temp>0:
            last=temp%10
            sum+=last
            product*=last
            temp=temp//10
        
        return product-sum

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
        int temp=n;
        int sum=0;
        int product=1;
        while(temp>0){
            int last=temp%10;
            sum+=last;
            product*=last;
            temp=temp/10;
        }
        
        return product-sum;
    }
};